elyra/contents/handlers.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+
+from jupyter_server.base.handlers import APIHandler
+from jupyter_server.utils import url_unescape
+from tornado import web
+
+from elyra.contents import ContentParser
+from elyra.util import get_expanded_path, get_absolute_path
+from elyra.util.http import HttpErrorMixin
+
+
+class FileParserHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose REST API to parse envs from a File"""
+    contentParser: ContentParser = ContentParser()
```

Comment:
Should probably use pythonic syntax for names here.
```suggestion
    content_parser: ContentParser = ContentParser()
```